### PR TITLE
Dedupe and check dupes

### PR DESCRIPTION
**Pull request by michal-wknd**
I added 2 features:
- check whether the same definition already exists before saving a new definition
- dedupe definitions for all acronyms

Fyi, I was not sure how to test this without adding it to my Slack workspace. Is it possible to do that? Thank you for reviewing this PR!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 1. npm install
 2. download ngrok https://ngrok.com/download
-3. ./ngrok http 8000
+3. ngrok http 8000
```
Comment (michal-wknd) [inline README.md:7]: changed this because it did not get installed in the project itself

File: README.md [hunk 2]
```diff
@@ -33,5 +35,5 @@ You can also create multiple definitions in a single Slack message by writing ea
 - [x] Save the definitions to a database so they persist
 - [x] Post to a channel when the definition does not exist so people can be notified to define it
 - [x] Post to a channel when something is defined so people can add or remove definitions
-- [ ] Add ability to remove definitions
+- [x] Add ability to remove definitions
```
Comment (michal-wknd) [inline README.md:38]: checked this box because this functionality is already there

File: utils.js [hunk 3]
```diff
@@ -101,10 +108,17 @@ const postDefinition = (res, acronym, definitions, channel_id, username, token)
...
 
 const newDefinition = (res, acronym, definition, channel_id, token, username, app_id) => {
-  saveAcronym(acronym, definition, app_id)
-  postToChannel(WAS_DEFINED_MESSAGE(acronym, username, definition), channel_id, token)
-  addOneTransaction('acronymsDefined')
-  res.json({ text: THANK_YOU_MESSAGE(acronym) })
+  // Checking whether the exact definition for this acronym already exists first
+  const definitionSnapshot = await getDefinition(acronym, definition, app_id).once('value')
+  const definitionFromDB = definitionSnapshot.val()
+  if (definitionFromDB.toLowerCase() === definition.toLowerCase()) {
```
Comment (michal-wknd) [inline utils.js:114]: did not check for italic, bold, etc. - unless a library exists to check all of those I think it would be complicated to implement.

**Review by DanielAndrews43 — APPROVED** [review]

**Comment by DanielAndrews43** [comment]
nevermind, this crashes

**Comment by michal-wknd** [comment]
> nevermind, this crashes

Oops sorry about that, i’ll take a look when i get a chance.

**Pull request merged**